docs/en/observability/observability-ui.asciidoc

```diff
@@ -3,10 +3,23 @@
 = View data in {kib}
 
 ++++
-<titleabbrev>View data</titleabbrev>
+<titleabbrev>Analyst overview page</titleabbrev>
```

Comment:
The abbreviated title seems too different from the actual title. It's a bit jarring. 

Also, this topic seems out of place since it's the only place where we provide a sort of "tour" of the UI. I wonder if it would be possible to eventually remove this page and work more guidance into the UI itself?

Reply:
Good point. I agree, eventually I think this page can be removed as we add more content to the overall guide. For the time being, I'll rename the heading and the abbreviated title to the **Observability overview page**.